redfish-core/lib/memory.hpp

```diff
@@ -809,6 +871,33 @@ inline void getDimmData(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
         std::bind_front(handleGetDimmData, asyncResp, dimmId));
 }
 
+/**
+ * @brief API used to process the Memory "Enabled" member which is
+ *        patched to do appropriate action.
+ *
+ * @param[in] resp - The redfish response to return.
+ * @param[in] dimmId - The patched Memory (aka DIMM) resource id.
+ * @param[in] enabled - The patched "Enabled" member value.
+ *
+ * @return The redfish response in the given buffer.
+ *
+ * @note - The "Enabled" member of the Memory (aka DIMM) is used to enable
+ *         (aka isolate) or disable (aka deisolate) the resource from the
+ *         system boot so this function will call "processHardwareIsolationReq"
+ *         function which is used to handle the resource isolation request.
+ *       - The "Enabled" member of the Memory is mapped with
+ *         "xyz.openbmc_project.Object.Enable::Enabled" dbus property.
+ */
+
+inline void patchMemberEnabled(const std::shared_ptr<bmcweb::AsyncResp>& resp,
```

Comment:
Please use `asynResp` for all occurrences

Reply:
done